recipe_scrapers/_exceptions.py

```diff
@@ -42,3 +42,10 @@ class SchemaOrgException(RecipeScrapersExceptions):
 
     def __init__(self, message):
         super().__init__(message)
+
+
+class ScrapingNotAllowed(RecipeScrapersExceptions):
+    def __init__(self, host_name):
+        self.host_name = host_name
+        message = f"Scraping not allowed at ({self.host_name})."
```

Comment:
We don't know whether an application developer or the end user will see and/or interpret this message, but I think it may be worth adding some more context.

How about: `Automated clients are not currently allowed according to the robots.txt policy for {self.host_name}`

In many cases I think that the application is _not_ going to be an automatic one -- it could be a command-line tool, or a web interface where a user provides a URL to be viewed (I think plaintextrecipes is a bit like that, if I remember correctly).  And that's not the intended audience of the [`robots.txt` spec in RFC9309](https://www.rfc-editor.org/rfc/rfc9309.html#name-introduction), so we should try to make clear that this is really only to prevent automated retrievals of the kind that can cause problems for site operators.

Reply:
(if the above seems reasonable: then that'd probably imply that we should also update the name of the exception class)